test

```diff
@@ -1,3 +1,4 @@
 Dies ist Zeile 1: Hallo, Welt!
 Dies ist Zeile 2: Hallo, Venus!
-Dies ist Zeile 3: Hallo, Rest des Universums!
+Dies ist Zeile 2.5: Hallo nichts!
+Dies ist Zeile 3: Hallo, Erde!
```

Comment:
Du bringst hier total die Reihenfolge durcheinander. Wir haben Erde doch schon in der ersten Zeile!